src/server/test/web/readingsLineMeterRangeRaw.js

```diff
@@ -93,7 +93,94 @@ mocha.describe('readings API', () => {
                     });
 
                     // Add LR14 here
+                    mocha.it('LR14: range should have daily points for 15 minute reading intervals aawnd r units with +-inf start/end time & C as F with intercept', async () => {
```

Comment:
Test description appears to have a typo. Should read as:
```suggestion
                    mocha.it('LR14:range should have daily points for 15 minute reading intervals and raw units with +-inf start/end time & C as F with intercept', async () => {
```